ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -27,18 +27,28 @@ open class SegmentedControl: UIView, TokenizedControlInternal {
 
     @objc public var isAnimated: Bool = true
     @objc public var numberOfSegments: Int { return items.count }
-    @objc public var shouldSetEqualWidthForSegments: Bool = true
+    /// Defines if the segments of the `SegmentedControl` are equal in width.
+    @objc public var shouldSetEqualWidthForSegments: Bool = true {
```

Comment:
it will be less intrusive if this was optional datasource

Reply:
I can see an argument for making `isFixedWidth` optional, and returning the value of `shouldSetEqualWidthForSegments` if it's nil, but could you help me understand how making `shouldSetEqualWidthForSegments` optional would be better? It wasn't optional before, and the default behavior hasn't changed, so I think making it optional would just add a third state.